systemtest/src/main/java/io/strimzi/systemtest/utils/ClientUtils.java

```diff
@@ -50,14 +50,13 @@ public static void waitUntilClientReceivedMessagesTls(KafkaClientOperations kafk
     public static void waitTillContinuousClientsFinish(String producerName, String consumerName, String namespace, int messageCount) {
         LOGGER.info("Waiting till producer {} and consumer {} finish", producerName, consumerName);
         TestUtils.waitFor("continuous clients finished", Constants.GLOBAL_POLL_INTERVAL, timeoutForClientFinishJob(messageCount),
-            () -> kubeClient().getClient().batch().jobs().inNamespace(namespace).withName(producerName).get().getStatus().getSucceeded().equals(1) &&
-                    kubeClient().getClient().batch().jobs().inNamespace(namespace).withName(consumerName).get().getStatus().getSucceeded().equals(1));
+            () -> kubeClient().getJobStatus(producerName, namespace) && kubeClient().getJobStatus(consumerName, namespace));
     }
 
     public static void waitForClientSuccess(String jobName, String namespace, int messageCount) {
         LOGGER.info("Waiting for producer/consumer:{} will be finished", jobName);
         TestUtils.waitFor("job finished", Constants.GLOBAL_POLL_INTERVAL, timeoutForClientFinishJob(messageCount),
-            () -> kubeClient().getClient().batch().jobs().inNamespace(namespace).withName(jobName).get().getStatus().getSucceeded().equals(1));
+            () -> kubeClient().getJobStatus(jobName, namespace));
```

Comment:
I would prepare to use the same approach as we have for deployment. Use current namespace if not specified and in case it's specified, use different.

Reply:
You mean by specifying it in the `kubeClient()` right? Like we have -> `kubeClient("some-namespace")`. I totally forgot this! Thanks :)